todo/views.py

```diff
@@ -0,0 +1,17 @@
+from .models import Todo
+from .serializers import TodoSerializer
+from rest_framework import mixins
+from rest_framework import generics
+
+class SnippetList(mixins.ListModelMixin,
```

Comment:
Please follow the standard naming convention 
`SnippetList` viewset is only responsible for listing 
and it should be `TodoList`